FhirToDataLake/deploy/templates/DeployAzureContainerApp.json

```diff
@@ -180,7 +202,7 @@
         "sku": {
           "name": "PerGB2018"
         },
-        "retentionInDays": 30
+        "retentionInDays": 60
```

Comment:
What's the purpose of the retentionInDays here?

Reply:
This is the retention time for logs in Log Analytics Workspace